packages/block-editor/src/components/discover-block-info/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { Icon } from '@wordpress/components';
+import { _n, sprintf } from '@wordpress/i18n';
+
+function DiscoverBlockInfo( { description, activeInstalls, humanizedUpdated } ) {
+	return (
+		<Fragment>
+			<span className="block-editor-discover-block-info__content">
+				{ description }
+			</span>
+			<div className="block-editor-discover-block-info__row">
+				<div className="block-editor-discover-block-info__column">
+					<Icon icon={ 'chart-line' }></Icon>{ sprintf( _n( '%d active installation', '%d active installations', activeInstalls ), activeInstalls ) }
```

Comment:
Here and in other places, you don't need to use evaluation for strings, you can pass them like with a regular HTML attribute:
```suggestion
					<Icon icon="chart-line"></Icon>{ sprintf( _n( '%d active installation', '%d active installations', activeInstalls ), activeInstalls ) }
```